vars/packageBuildingPipelineDAOS.groovy

```diff
@@ -631,11 +631,14 @@ def call(Map pipeline_args) {
                                           git checkout ''' + pipeline_args.get('daos_test_branch',
                                                                                'origin/master') + '''
                                           git submodule update --init
-                                          make PR_REPOS="''' + env.JOB_NAME.split('/')[1] +
+                                          if make PR_REPOS="''' + env.JOB_NAME.split('/')[1] +
                                                '@' + env.BRANCH_NAME + ':' + env.BUILD_NUMBER +
                                                ' ' + pipeline_args.get('test_repos', '') + '''" \
                                                CHROOT_NAME=epel-7-x86_64                        \
-                                               -C utils/rpms chrootbuild'''
+                                               -C utils/rpms chrootbuild; then
+                                            grep 'No matching package to install'               \
```

Comment:
I'm concerned about false positives here, where a package should be found but isn't due to a bug.

I understand the problem we are trying to solve, but just wonder if there is an alternate way to solve it.

What about global Jenkins variable called BOOTSTRAP (or something aptly named) which allows this stage to fail, or not even be run perhaps.

Reply:
A missing package other than the package being built is not something that should block a PR for or a build of that package.

Right now, I need to run a new set of builds on the master branch, and this test is preventing it because the other RPMs are not present.

Reply:
It should be if it's got a `Requires:` for something that cannot be found because the `Requires:` is wrong.

Reply:
A more comprehensive sanity check that still allows a bootstrap condition can be added later.

This should be good enough to get the packages built.